dashboards/blazor-web-assembly/blazor-web-assembly.json

```diff
@@ -0,0 +1,1158 @@
+			"description": null,
+			"widgets": [
+				{
+					"title": "",
+					"layout": {
+						"column": 1,
+						"row": 1,
+						"width": 2,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.markdown"
+					},
+					"rawConfiguration": {
+						"text": "![NuxtJS icon](https://www.zenesys.com/getattachment/Blog/Web-Development/January-2022/Blazor-Server-vs-Blazor-Web-Assembly/MicrosoftTeams-image-(15).png?width=400&height=300)\n"
+					}
+				},
+				{
+					"title": "Applications (click on the application  name to filter)",
+					"layout": {
+						"column": 3,
+						"row": 1,
+						"width": 3,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.bar"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountId": 0,
+								"query": "FROM PageView SELECT count(*) FACET appName LIMIT MAX "
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "List of Page URLs (click on the page URL to filter)",
+					"layout": {
+						"column": 6,
+						"row": 1,
+						"width": 4,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.bar"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountId": 0,
+								"query": "SELECT count(*) FROM PageView FACET pageUrl"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "Browser Names (click on the browser name to filter)",
+					"layout": {
+						"column": 10,
+						"row": 1,
+						"width": 3,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.bar"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountId": 0,
+								"query": "SELECT count(*) as 'Page Views' FROM PageView FACET userAgentName"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "",
+					"layout": {
+						"column": 1,
+						"row": 4,
+						"width": 2,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.markdown"
+					},
+					"rawConfiguration": {
+						"text": "**About**\n\nInstrument your application with New Relic - [Add Data](https://docs.newrelic.com/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/).\n\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=BlazorWebAssembly) here and let us know how we can improve it for you."
```

Comment:
please update the "Add Data" link to "https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/browser-monitoring-integrations/blazor-webassembly/"

Reply:
Hi @pkosanam,
Updated doc's URL in the dashboard. Kindly review it